flipperdevices/Flipper-Android-App
# Remotecontrols/design fixes

**Background**
Remote controls has some design issues and lack of favorite option

**Changes**

- Fixed text on setup screen
- Add favorite option to saved remote

**Test plan**

- Open setup screen and see new texts description
- See saved remote and try to favorite it
